srcpkgs/libsigrok/patches/0001-Change-plugdev-to-users.patch

```diff
@@ -0,0 +1,24 @@
+From 1cf31f45c0b4ac2ccd08827be589391393db9e58 Mon Sep 17 00:00:00 2001
+From: Jasper Chan <jasperchan515@gmail.com>
+Date: Mon, 27 Jul 2020 18:41:19 -0700
+Subject: [PATCH] Change plugdev to users
+
+---
+ contrib/61-libsigrok-plugdev.rules | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git contrib/61-libsigrok-plugdev.rules contrib/61-libsigrok-plugdev.rules
+index 2fddba48..ff03a8ee 100644
+--- contrib/61-libsigrok-plugdev.rules
++++ contrib/61-libsigrok-plugdev.rules
+@@ -26,6 +26,6 @@
+ 
+ ACTION!="add|change", GOTO="libsigrok_rules_plugdev_end"
+ 
+-ENV{ID_SIGROK}=="1", MODE="660", GROUP="plugdev"
++ENV{ID_SIGROK}=="1", MODE="660", GROUP="users"
```

Comment:
For what it's worth, `openocd` has a `plugdev` group. Maybe we could add that or make a small package that only adds the `plugdev` group? And then everything that uses it can depend on the plugdev meta-package.